moped-editor/src/views/projects/projectView/ProjectComponents/utils/useUpdateComponent.js

```diff
@@ -37,6 +37,7 @@ const editReducer = (state, action) => {
         ...state,
         showEditModeDialog: false,
         isEditingComponent: false,
+        draftEditComponent: null,
```

Comment:
we do not want a draftEditComponent if we're not editing.  when the draftEditComponent is not null, that specific component gets filtered out of the project lines and points 
layers [here](https://github.com/cityofaustin/atd-moped/blob/7ad962c621f708e1e0d6b50e872b890b8f7e1490/moped-editor/src/views/projects/projectView/ProjectComponents/ProjectSourcesAndLayers.js#L28-L44).